mongo/client.go

```diff
@@ -275,7 +275,8 @@ func (c *Client) StartSession(opts ...*options.SessionOptions) (Session, error)
 		return nil, replaceErrors(err)
 	}
 
-	sess.RetryWrite = c.retryWrites
+	// Writes are not retryable on standalones, so let operation determine whether to retry
+	sess.RetryWrite = false
```

Comment:
Hmm, can we remove `RetryWrite` and `RetryRead` from the `session.Client` struct? IIUC the decision for whether an operation can be retried comes from the `client` struct's `retryWrites` and `retryReads` fields.

If that is the case, I can create another ticket to remove it if it is not straightforward. If it is straightforward, I'd vote for removing it as part of this ticket.

Reply:
I'm not sure we can, because operation.Client is actually a session.Client, which is what it uses to determine retryabliity.

Reply:
Ah I missed that. I think I assumed `operation.Client` was a `mongo.Client`.